mysql-test/mroonga/include/mroonga/check_version.inc

```diff
@@ -15,28 +15,32 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 
 --disable_query_log
-let $version_major_minor =
-  `SELECT CAST(SUBSTRING_INDEX(@@global.version, '.', 2) AS DECIMAL(4, 2))`;
```

Comment:
Why do we need these changes?
Is it for distinguishing `10.1` and `10.10`? Then how about the following?

```sql
SELECT CAST(
         CONCAT(
           SUBSTRING_INDEX(@@global.version, '.', 1),
           '.',
           LPAD(
             SUBSTRING_INDEX(
               SUBSTRING_INDEX(@@global.version, '.', 2),
               '.',
               -1
             ),
             2,
             '0'
           )
         )
         AS DECIMAL(4, 2)
       );
```

Reply:
> Is it for distinguishing 10.1 and 10.10? Then how about the following?

Yes, currently 10.10 is regarded as 10.1, and tests failed by that reason.
Tests that should be skipped greater or equals to 10.5 are executed because 10.10 is regarded as 10.1.

Reply:
I will try this SQL!